umap/static/umap/js/modules/schema.js

```diff
@@ -1,45 +1,52 @@
 import { translate } from './i18n.js'
 
+// Possible impacts
+// ['ui', 'data', 'limit-bounds', 'datalayer-index', 'remote-data', 'background']
+
 export const SCHEMA = {
-  zoom: {
-    type: Number,
-  },
-  scrollWheelZoom: {
+  browsable: {
     type: Boolean,
-    label: translate('Allow scroll wheel zoom?'),
+    impacts: ['ui'],
   },
-  scaleControl: {
+  captionBar: {
+    impacts: ['ui'],
     type: Boolean,
-    label: translate('Do you want to display the scale control?'),
-    default: true,
+    label: translate('Do you want to display a caption bar?'),
+    default: false,
+    impacts: [''],
   },
-  moreControl: {
+  captionMenus: {
+    impacts: ['ui'],
     type: Boolean,
-    label: translate('Do you want to display the «more» control?'),
+    label: translate('Do you want to display caption menus?'),
     default: true,
   },
-  miniMap: {
-    type: Boolean,
-    label: translate('Do you want to display a minimap?'),
-    default: false,
-  },
-  displayPopupFooter: {
-    type: Boolean,
-    label: translate('Do you want to display popup footer?'),
-    default: false,
+  color: {
+    impacts: ['data'],
+    type: String,
+    handler: 'ColorPicker',
+    label: translate('color'),
+    helpEntries: 'colorValue',
+    inheritable: true,
+    default: 'DarkBlue',
   },
-  onLoadPanel: {
+  dashArray: {
+    impacts: ['data'],
     type: String,
-    label: translate('Do you want to display a panel on load?'),
-    choices: [
-      ['none', translate('None')],
-      ['caption', translate('Caption')],
-      ['databrowser', translate('Data browser')],
-      ['facet', translate('Facet search')],
-    ],
-    default: 'none',
+    label: translate('dash array'),
+    helpEntries: 'dashArray',
+    inheritable: true,
+  },
+  datalayersControl: {
+    impacts: ['ui'],
+    type: Boolean,
+    nullable: true,
+    handler: 'DataLayersControl',
+    label: translate('Display the data layers control'),
+    default: true,
   },
   defaultView: {
+    impacts: [], // no need to update the ui, only useful when loading the map
```

Comment:
Is there a difference between `['']` (above) and `[]`? Maybe update the `// Possible impacts` comment to explain that?

Reply:
No, there shouldn't be, and the list should be updated to only have `[]`.